cfgov/v1/models/base.py

```diff
@@ -58,15 +53,6 @@ def shared_q(self):
     def shared(self):
```

Comment:
Can `shared` and `shared_q` be cleaned up? And, if so, can this entire custom `CFGOVPageQuerySet` class be removed?